filters/quantitative_ques/filter.py

```diff
@@ -0,0 +1,33 @@
+import spacy
+from interfaces.QuestionAnswerOperation import QuestionAnswerOperation
+from tasks.TaskTypes import TaskType
+
+
+class QuantitativeQuestion(QuestionAnswerOperation):
```

Comment:
It would be great if we could add a few lines of comments for this class.
That will be useful to have a quick overview of this filter, without going through the code.